### PR TITLE
fix exemption display bug

### PR DESCRIPTION
**Pull request by tinglisD2L**

### CODE REVIEW

File: utility/completion-status-mixin.html [hunk 1]
```diff
@@ -61,12 +61,17 @@
 
 			_getCompletionRequirement(activity) {
 				var itemSubEntity = activity.getSubEntityByRel('item');
+				var aboutSubEntity =  activity.getSubEntityByRel('about');
```
Comment (lawong-d2l) [inline utility/completion-status-mixin.html]: Do we know why sometime it's 'about' and other times it's 'item'? This seems like the root of the issue. I feel like fixing it here is us hiding something else.

Comment (tinglisD2L) [reply utility/completion-status-mixin.html]: ok now it's calling by class rather than rel. i think it's better.

Comment (tinglisD2L) [reply utility/completion-status-mixin.html]: it also fixed the problem of some "Optional" labels not showing. I didn't realize it before!

File: utility/completion-status-mixin.html [hunk 2]
```diff
@@ -60,29 +60,28 @@
 			}
 
 			_getCompletionRequirement(activity) {
-				var itemSubEntity = activity.getSubEntityByRel('item');
-				if (itemSubEntity
-					&& itemSubEntity.getSubEntityByClass('exemption')
-					&& itemSubEntity.getSubEntityByClass('exemption').hasClass('exempted')) {
+				var subEntity = activity.getSubEntityByClass('link-activity') || activity.getSubEntityByClass('file-activity');
```
Comment (lawong-d2l) [inline utility/completion-status-mixin.html]: Are we sure that these two are the only possible classes we need to be looking out for?

**Pull request merged**